.mergify.yml

```diff
@@ -43,13 +43,26 @@ queue_rules:
               - "status-success=ci/centos/jjb-validate"
 
 pull_request_rules:
+  - name: start CI jobs for queued PR
+    conditions:
+      - base~=^(devel)|(release-.+)$
+      - "check-pending=Queue: Embarked in merge train"
```

Comment:
This will get added when we have the required approvals and only on one PR, which is in queue, right? Not on all the PRs? If it gets added to all the PRs in the queue again, its waste of resource

Reply:
I am not sure if it gets added to all PRs, that is difficult to verify without testing it out first.

Reply:
Lets merge this and verify it out :dagger: